src/Modules/Subscription.php

```diff
@@ -58,21 +62,23 @@ public function __construct(
 
         // Make request and set response data as properties
         $this->setResponse(\Mobbex\Api::request([
-            'uri'    => 'subscriptions/' . $uid,
             'method' => 'POST',
+            'uri'    => 'subscriptions/' . $uid,
             'body'   => \Mobbex\Platform::hook($hookName, true, [
-                'total'       => $total,
-                'type'        => $type,
-                'webhook'     => $webhookUrl,
-                'return_url'  => $returnUrl,
                 'currency'    => 'ARS',
-                'reference'   => $this->reference = \Mobbex\Platform::$name . '_id:' . $id,
+                'test'        => $test,
```

Comment:
El test no debería tomarlo de `$this->settings['test']` en lugar de pasarlo como parámetro?